rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -1173,6 +1175,17 @@ stream {
             proxy_set_header       X-Service-Port     $service_port;
             {{ end }}
 
+            {{/* if a location-specific error override is set, add the proxy_intercept here */}}
+            {{ if $location.CustomHTTPErrors }}
+            # Custom error pages per ingress
+            proxy_intercept_errors on;
+            {{ end }}
+
+            {{ range $errCode := $location.CustomHTTPErrors }}
+            error_page {{ $errCode }} = @custom_{{ $errCode }};{{ end }}
+
+
+            {{ if not (empty $location.Backend) }}
```

Comment:
you need to remove this line (this is the reason of the error in ci)

Reply:
ah good catch, thanks!